src/compiler/scanner.ts

```diff
@@ -364,6 +365,11 @@ namespace ts {
 
     const hasOwnProperty = Object.prototype.hasOwnProperty;
 
+    export function isWhiteSpaceLike(ch: number): boolean {
+        return isWhiteSpace(ch) || isLineBreak(ch);
+    }
+
+    /** Does not include line breaks. For that, see isWhiteSpaceLike. */

```

Comment:
is there some terminology we could borrow from regexes for this? I know regexes have a distinction between dot-matches-newline and dot-doesn't-match-newline.